tests/test_view_model.py

```diff
@@ -25,7 +25,7 @@ def test_done_items_when_done_item_exists():
     
     assert len(view_model.items) == 2
     assert len(view_model.done_items) == 1
-    assert view_model.done_items == [Item(2, 'Task 2', datetime.now(), DONE_LIST_NAME)]
+    assert all(view_model.done_items) == all([Item(2, 'Task 2', datetime.now(), DONE_LIST_NAME)])
```

Comment:
I don't think this is doing what you're hoping for:
> The all() function returns True if all items in an iterable are true, otherwise it returns False.

[docs](https://www.w3schools.com/python/ref_func_all.asp)

So this is just checking that each element is "truthy" and then checking that "True == True"

If I break your viewmodel so that todo actually filters for done and vice versa, this test can still pass

Reply:
i've changed this